javascript/javascript2/week2/movies.js

```diff
@@ -0,0 +1,193 @@
+const movies = [
```

Comment:
For the movies exercise, please not include all the movies array. is not okay...you should not make your own movies variable. Instead include javascript2/week2/homework/movies.js into your .html file

Reply:
Hi Florin,
I tried several ways to include movies array from external file but it isnt working.... but I still have this modified vesion with external movies array.